network-api/networkapi/buyersguide/templates/product_page.html

```diff
@@ -11,12 +11,21 @@
 
 {% block guts %}
 <div class="container-fluid text-center product-header bg-gray">
-  <img class="mb-0" src="{{mediaUrl}}{{"AWS_LOCATION"|env}}/{{product.image}}" width="400" alt="{{product.name}}"/>
+  <div class="container">
+    <div class="row">
+      <div class="col-12 col-md-10 offset-md-1">
+        <img class="mb-0" src="{{mediaUrl}}{{"AWS_LOCATION"|env}}/{{product.image}}" width="400" alt="{{product.name}}"/>
+        {% if product.meets_minimum_security_standards %}
+        <img src="/_images/buyers-guide/badge.svg" class="seal-of-approval" alt="this product meets minimum security standards" />
```

Comment:
Do we want to be more descriptive in this alt text?
```suggestion
        <img src="/_images/buyers-guide/badge.svg" class="seal-of-approval" alt="this product meets Mozilla's minimum security standards" />
```
/cc @kristinashu

Reply:
I'm gonna go with `This product meets our minimum security standards` since that's what it says in the seal